README.md

```diff
@@ -50,7 +50,9 @@ _Resulting layout:_
 ```html
 <header>
   <h1>Blog 2000</h1>
-  <h2>All articles</h2>
+  <yield to="header">
```

Comment:
Hmm, I wasn't really sure what to call that attribute, to be honest. `target` could work, I suppose. However, I feel like it should read as if it was said out loud. “Yield to toolbar” or “yield target toolbar” are both okay, I suppose. Or maybe even `yield` is wrong. As you say, that's heavily influenced by Rails, but we're at liberty to call it whatever we want. :smiley: